Gems/GradientSignal/Code/Include/GradientSignal/PerlinImprovedNoise.h

```diff
@@ -27,6 +27,11 @@ namespace GradientSignal
         */          
         PerlinImprovedNoise(int seed);
 
+        /**
+         * Initializes the permutation table from a preset table
+         */
+        PerlinImprovedNoise(const AZStd::array<int, 512>& permutationTable);
```

Comment:
How about accepting an `AZStd::span<int, 512>`?
Yes a span can accept a maximum extent value as it's second [template parameter](https://en.cppreference.com/w/cpp/container/span)